app/controllers/recipes_controller.rb

```diff
@@ -31,6 +31,11 @@ def destroy
     redirect_to recipes_path
   end
 
+  def update
+    @recipe = Recipe.find_by(id: params[:id])
+    @recipe.update_attribute(:public, false)
+  end
```

Comment:
Shouldn't we toggle the public attribute? I mean, the user should be able to return to `true` value of the `public` attribute.
I think we should do:
```Ruby
public = params[:public] == '1'
@recipe.update_attribute(:public, public)
```